app/views/support_interface/application_forms/courses/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :title, 'Pick a course' %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_search_course_new_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @pick_course, url: support_interface_application_form_create_course_path(course_code: @course_code) do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.govuk_radio_buttons_fieldset :course_option_id, legend:  { text: "Search results for course code '#{@course_code}'", size: "m" } do %>
```

Comment:
This legend might need to be more directly related to the purpose of the form, for accessibility reasons?

Could have a preceding heading which includes the search term?

```suggestion
      <%= f.govuk_radio_buttons_fieldset :course_option_id, legend:  { text: "Which course should be added to the application?", size: "m" } do %>
```